consumers/best-practices.md

```diff
@@ -1,26 +1,39 @@
 ---
-description: How to use Pyth's Price Feeds.
+description: Introduction to Price Feeds
 ---
 
-# Fixed-Point Representation
+This page provides some technical details about Pyth price feeds that are necessary to use them safely and correctly.
 
```

Comment:
I think it might be worth saying something like "Please read this before using Pyth price feeds."